app/views/customers/passwords/edit.html.slim

```diff
@@ -0,0 +1,14 @@
+.signup-form-wrapper
+  h2 Change your password
+  = simple_form_for(resource, as: resource_name, url: customer_password_path, html: { method: :put }) do |f|
+    - if resource.errors[:reset_password_token].present?
+      = flash_container(BootstrapFlashHelper::ALERT_TYPES_MAP.fetch(:alert), 'Reset password token has expired, please request a new one')
```

Comment:
Why you hardcode error message about `reset_password_token` here? Why not use error that you receive from API?